README.md

```diff
@@ -1,2 +1,68 @@
-# Ovio.org
-Updated website for [Ovio.org](ovio.org)
+# Ovio website
+
+https://oviohub.com/
+
+A GitHub repo for Ovio static website.
+
+## Development
+
+### Getting started
+
+This project use **[yarn](https://yarnpkg.com/en/)** package manager with **[gatsbyJs](https://www.gatsbyjs.org/)**, a project builder for **[React](https://reactjs.org/)**.
+
+```bash
+yarn
+yarn start
+```
+The website will be available at <http://localhost:8000/>.
+
+### Deployement
+
+To generate the ```public``` folder, run
+
+```bash
+yarn build
+```
+
+### Project structure
+
+```
+website/
+│   README.md
+│   gatsby-config.js               <-- metadata for project title, description, plugins, etc.
+│   gatsby-node.js                 <-- generation of pages in src/Routes.js
+│   package.json                   <-- configuration of the project (dependencies, scripts, ...)
+│   yarn.lock                      <-- auto-generated by yarn to manage dependencies
+│
+└───public/                        <-- auto-generated on "yarn start" command run
+│   │   ....
+|
+└───node_modules/                  <-- auto-generated by yarn to manage dependencies
+│   │   ....
+│
+└───src/
+│   │   Routes.js                  <-- contains informations (url, seo params, ...) of website's pages
```

Comment:
the word "information" is always singular, not plural. We typically use "about" instead of "of" in this context too. 

So it would be "contains information (...) about the website's pages"

English is weird ;-)

Reply:
Copy that 

No weird as French :D